api/server/router/container/container_routes.go

```diff
@@ -446,10 +428,6 @@ func (s *containerRouter) wsContainersAttach(ctx context.Context, w http.Respons
 	}
 	containerName := vars["name"]
 
-	if !s.backend.Exists(containerName) {
-		return derr.ErrorCodeNoSuchContainer.WithArgs(containerName)
```

Comment:
This is what is causing the websockets test failure. It fails inside the hijacked stream, vs before hijacking.